### PR TITLE
#68 - Module Unification call for help / contributions

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: source/blog/2018-10-12-the-ember-times-issue-68.md [hunk 1]
```diff
@@ -52,8 +52,17 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification is in the final stretch!](https://github.com/emberjs/ember.js/issues/16373)
```
Comment (amyrlam) [inline source/blog/2018-10-12-the-ember-times-issue-68.md]: this is an awesome writeup! can you change to title case re: https://github.com/emberjs/website/blob/master/source/blog/CONTRIBUTING.md#writing-style-guide? check out https://titlecaseconverter.com/

File: source/blog/2018-10-12-the-ember-times-issue-68.md [hunk 2]
```diff
@@ -52,8 +52,17 @@ responsive: true
...
+Have you heard of **Ember's future default project layout**? If you haven't, you can read about it the [Module Unification RFC](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md). 
+
+The initial work on Module Unification is **nearly complete**, as **[there are only a handful of tasks left](https://github.com/emberjs/ember.js/issues/16373)** and some of the tasks just need a good amount of spelunking time.
+If you can donate some of your time to help finish this up, the entire community will forever be in your debt 💖
```
Comment (amyrlam) [inline source/blog/2018-10-12-the-ember-times-issue-68.md]: change to `debt. 💖`

File: source/blog/2018-10-12-the-ember-times-issue-68.md [hunk 3]
```diff
@@ -52,8 +52,17 @@ responsive: true
...
+
+Also, if you are a user of namespaced components, [this RFC](https://github.com/emberjs/rfcs/pull/367) is a must read. Please leave your thoughts on the [Module Unification Packages RFC](https://github.com/emberjs/rfcs/pull/367) as nested components affect the majority of Ember projects.
+
+To learn more about Module Unification, feel free to join the #st-module-unification channel on the [Ember Discord](https://discordapp.com/invite/zT3asNS)!
```
Comment (amyrlam) [inline source/blog/2018-10-12-the-ember-times-issue-68.md]: i recently learned that you can link to a Discord channel on the web. so could change to `[#st-module-unification](https://discordapp.com/channels/480462759797063690/484527343331704832)`

File: source/blog/2018-10-12-the-ember-times-issue-68.md [hunk 4]
```diff
@@ -52,8 +52,17 @@ responsive: true
...
 
+Curious what a real project looks like with module unifcation? Checkout [emberclear](https://emberclear.io) by [@NullVoxPopuli](http://github.com/nullvoxPopuli/) on [gitlab](https://gitlab.com/NullVoxPopuli/emberclear/tree/master/packages/frontend) or [github](https://github.com/NullVoxPopuli/emberclear/tree/master/packages/frontend)!
```
Comment (amyrlam) [inline source/blog/2018-10-12-the-ember-times-issue-68.md]: sorry one last comment, this is rad, i love example repos, really helps me learn

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 1 above)
Comment (NullVoxPopuli) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: hmm, it did `Module Unification Is in the Final Stretch`
wouldn't `Module Unification is in the Final Stretch` be more correct? 
(lowercase `is`)

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 3 above)
Comment (NullVoxPopuli) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: silly native app

Comment (NullVoxPopuli) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: should the invite link be kept? or no?

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 4 above)
Comment (NullVoxPopuli) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: <3 yay <3

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 1 above)
Comment (amyrlam) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: I would say just go with what it says to be consistent, up to you though, nbd!

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 3 above)
Comment (amyrlam) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: i say keep both to encourage the discord to be moar active, cool?

File: source/blog/2018-10-12-the-ember-times-issue-68.md [hunk 5]
```diff
@@ -52,8 +52,17 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification is in the Final Stretch!](https://github.com/emberjs/ember.js/issues/16373)
```
Comment (chrisrng) [inline source/blog/2018-10-12-the-ember-times-issue-68.md]: Sorry can we put an emoji here? 😄

Comment (NullVoxPopuli) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: Sure! Which one?

Comment (chrisrng) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: Author's pick haha Maybe 🛣 ?

**Comment by amyrlam** [comment]
@NullVoxPopuli we tend to just put our real names a la https://emberjs.com/blog/2018/10/05/the-ember-times-issue-67.html, inconsistent with the rest but more blog-friendly(?)

File: source/blog/2018-10-12-the-ember-times-issue-68.md (hunk 5 above)
Comment (amyrlam) [reply source/blog/2018-10-12-the-ember-times-issue-68.md]: yes good call, thx @chrisrng! 📈

**Comment by NullVoxPopuli** [comment]
ah ok, can do!

**Pull request merged**